plugins/Morpheus/templates/dashboard.twig

```diff
@@ -20,13 +20,13 @@
     {% include "@CoreHome/_warningInvalidHost.twig" %}
     {% include "@CoreHome/_topScreen.twig" %}
 
-    <div class="top_controls">
-        {% if hideTopControls is not defined or not hideTopControls %}
+    <div class="top_controls{% if allWebsitesDashboard|default(false) %} top_controls--awd{% endif %}">
+        {% if allWebsitesDashboard is not defined or not allWebsitesDashboard %}
```

Comment:
I think using something like `hideTopControls` suited better, than creating a logical dependency to the all websites dashboard.
Also instead of adding a new class, you could simply let the all websites dashboard set a body id, so you can use that in css. This should work by using `{% set bodyId = 'awd' %}` in the all websites template.

Reply:
We're not hiding top controls though.. we're just adjusting it, but ok, I can change it back.

Thanks for the tip with body ID, will use that.

Reply:
I've used `hideQuickAccess` as that's what it's doing. The top controls block is always visible again (outside of the condition).